ruby_event_store/lib/ruby_event_store/mappers/protobuf.rb

```diff
@@ -1,8 +1,8 @@
 module RubyEventStore
   class Proto < RubyEventStore::Event
-    def initialize(event_id: SecureRandom.uuid, metadata: {}, data:)
+    def initialize(event_id: SecureRandom.uuid, metadata: nil, data:)
       @event_id = event_id
-      @metadata = metadata
+      @metadata = Metadata.new(metadata.to_h)
```

Comment:
Oh wow, today I learned that `nil.to_h` returns an empty hash. Huh.

Reply:
indeed :)